packages/squiggle-lang/__tests__/dist/Symbolic_test.ts

```diff
@@ -252,3 +258,72 @@ describe("Bernoulli", () => {
     );
   });
 });
+
+describe("Metalog", () => {
+  const dist = mkMetalog([5, 2]);
+  const testRange = 10;
+  const testSteps = 1000;
+  const step = (testRange * 2) / testSteps;
+  const iter = E_A_Floats.range(-testRange, testRange, testSteps);
+
+  test("CDF only grows", () => {
+    const cdfValues = iter.map((v) => dist.cdf(v));
+    expect(E_A.pairwise(cdfValues, (a, b) => a <= b).every((v) => v)).toBe(
+      true
+    );
+  });
+
+  test("Runs quickly", () => 
+    new Promise((resolve, reject) => {
+      const session = new Session();
+      session.connect();
+
+      session.post('Profiler.enable', () => {
+        session.post('Profiler.start', () => {
+          iter.forEach((v) => dist.pdf(v));
+          session.post('Profiler.stop', (err, {profile}) => {
+            fs.writeFileSync('./metalog_pdf.cpuprofile', JSON.stringify(profile));
+            resolve(undefined)
+      });
+    });
+  });
+  }), 1000);
+
+  test("CDF conforms to PDF", () => {
+    expect(
+      E_A.pairwise(
+        iter.map((a) => [dist.cdf(a), unpackResult(dist.pdf(a))]),
+        (a, b) => [a, b]
+      ).forEach(
+        ([[cdf, pdf], [cdf2]]) => expect(cdf2).toBeCloseTo(cdf + pdf * step, 1) // This seems surprisingly not accurate even though our implementation matches rmetalog almost perfectly
```

Comment:
I ran this test 1000 times with that precision tuning removed and it passed, maybe there was a bug that you've fixed?

Reply:
Oh yeah! I forgot. That was a relic from when our implementation was as bad as rmetalog's